### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Comment by jpkrohling** [comment]
This change is [Reviewable](https://reviewable.io/reviews/jaegertracing/jaeger-operator/112)

**Comment by jpkrohling** [comment]
@objectiser , could you review this, especially the "notable new features" and "breaking changes" parts?

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+*Breaking changes*
+
+* Change CRD to use lower camel case (https://github.com/jaegertracing/jaeger-operator/pull/87[#87])
+* Remove zipkin service (https://github.com/jaegertracing/jaeger-operator/pull/75[#75])
+
+*Full list of commits:*
+
+* Query base path should be used to configure correct path in ingress (https://github.com/jaegertracing/jaeger-operator/pull/108[#108])
+* Enable resources to be defined at top level and overridden at compone… (https://github.com/jaegertracing/jaeger-operator/pull/110[#110])
```
Comment (objectiser) [inline CHANGELOG.adoc]: Should this be in notable new features?

File: CHANGELOG.adoc [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+* Fix disable-oauth-proxy example (https://github.com/jaegertracing/jaeger-operator/pull/107[#107])
+* Add OAuth Proxy to UI when on OpenShift (https://github.com/jaegertracing/jaeger-operator/pull/100[#100])
+* Refactor common spec elements into a single struct with common proces… (https://github.com/jaegertracing/jaeger-operator/pull/105[#105])
+* Ensure 'make generate' has been executed when model changes are made (https://github.com/jaegertracing/jaeger-operator/pull/101[#101])
+* Enable top level annotations to be defined (https://github.com/jaegertracing/jaeger-operator/pull/97[#97])
```
Comment (objectiser) [inline CHANGELOG.adoc]: Notable feature?

File: CHANGELOG.adoc [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+* Update generated code and reverted change to 'all-in-one' in CRD (https://github.com/jaegertracing/jaeger-operator/pull/98[#98])
+* Support volumes and volumeMounts (https://github.com/jaegertracing/jaeger-operator/pull/82[#82])
+* Update readme to include info about storage options being located in … (https://github.com/jaegertracing/jaeger-operator/pull/96[#96])
+* Enable storage options to be filtered out based on specified storage … (https://github.com/jaegertracing/jaeger-operator/pull/94[#94])
+* Add support for OpenShift routes (https://github.com/jaegertracing/jaeger-operator/pull/93[#93])
+* Change CRD to use lower camel case (https://github.com/jaegertracing/jaeger-operator/pull/87[#87])
+* Factor out ingress from all-in-one and query, as common to both but i… (https://github.com/jaegertracing/jaeger-operator/pull/91[#91])
```
Comment (objectiser) [inline CHANGELOG.adoc]: Breaking change.

File: CHANGELOG.adoc [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+* Fix operator SDK version as master is too unpredicatable at the moment (https://github.com/jaegertracing/jaeger-operator/pull/92[#92])
+* Update generated file after new annotations field (https://github.com/jaegertracing/jaeger-operator/pull/90[#90])
+* Enable annotations to be specified with the deployable components (https://github.com/jaegertracing/jaeger-operator/pull/86[#86])
```
Comment (objectiser) [inline CHANGELOG.adoc]: Notable?

File: CHANGELOG.adoc [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+* Remove zipkin service (https://github.com/jaegertracing/jaeger-operator/pull/75[#75])
+* Add support for Cassandra create-schema job (https://github.com/jaegertracing/jaeger-operator/pull/71[#71])
```
Comment (objectiser) [inline CHANGELOG.adoc]: Notable?

Comment (jpkrohling) [reply CHANGELOG.adoc]: This one is already there

File: CHANGELOG.adoc [hunk 6]
```diff
@@ -0,0 +1,103 @@
+:toc:
+
+= Changelog
+
+== v1.7.1 (2018-13-11)
```
Comment (objectiser) [inline CHANGELOG.adoc]: As discussed, this could now be for 1.8.0.

File: CHANGELOG.adoc (hunk 5 above)
Comment (objectiser) [reply CHANGELOG.adoc]: don't know why I missed that

File: CHANGELOG.adoc (hunk 6 above)
Comment (jpkrohling) [reply CHANGELOG.adoc]: Done

**Pull request merged**